sentence_transformers/sparse_encoder/models/SpladePooling.py

```diff
@@ -28,14 +32,20 @@ class SpladePooling(Module):
                 - `relu`: ReLU activation (standard in all Splade models).
                 - `log1p_relu`: log(1 + ReLU(x)) variant used in Opensearch Splade models see arxiv.org/pdf/2504.14839.
         word_embedding_dimension (int, optional): Dimensionality of the output embeddings (if needed).
+        chunk_size (int, optional): Size of chunks when processing tokens. If None, processes entire sequence at once.
+            Using smaller chunks reduces memory usage but may impact speed performance. Default is None.
     """
 
     SPLADE_POOLING_MODES = ("sum", "max")
     SPLADE_ACTIVATION = ["relu", "log1p_relu"]
     config_keys: list[str] = ["pooling_strategy", "activation_function", "word_embedding_dimension"]
 
     def __init__(
-        self, pooling_strategy: str = "max", activation_function="relu", word_embedding_dimension: int = None
+        self,
+        pooling_strategy: str = "max",
```

Comment:
```suggestion
        pooling_strategy: Literal["max", "sum"] = "max",
```
This might be nice actually?